src/service/execution.ts

```diff
@@ -74,5 +74,11 @@ export async function executeOperation<T extends OperationContext>(
     await context.params.onFailed(result);
   }
 
+  if (result.error) {
+    logDb('Result:\n%O', result);
+
+    throw result.error;
+  }
+
   return result;
```

Comment:
Is this return used anywhere? @cham11ng

Reply:
@mesaugat I've updated the PR to explicitly commit on success and rollback on failure.

Reply:
There was a problem while explicitly committing in mssql may be it something in `tedious` but it worked in Postgres.
Reverted chages.

Reply:
> Is this return used anywhere?

On this part, I've updated the PR again and tested it locally.